RELEASE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Wikibase DataModel release notes
 
+## Version 5.0.2 (alpha)
```

Comment:
This is not new, as already argued in https://github.com/wmde/WikibaseDataModel/pull/626#issuecomment-185171983. The interfaces are untouched.
* `copy` behaves as before. The fact that immutable objects are not cloned any more can only have an impact on a user if he was using it wrong before.
* The possibility to use `clone` with Item and Property objects was there before, but not documented as part of the public interface of these classes. This did not changed.